test/modern_bpf/test_suites/syscall_enter_suite/access_e.cpp

```diff
@@ -0,0 +1,40 @@
+#include "../../event_class/event_class.h"
+
+#ifdef __NR_access
+
+TEST(SyscallEnter, accessE)
+{
+	auto evt_test = get_syscall_event_test(__NR_access, ENTER_EVENT);
+
+	evt_test->enable_capture();
+
+	/*=============================== TRIGGER SYSCALL  ===========================*/
+
+	int32_t _mode = W_OK;
+	const char *_pathname = "./mock_file";
```

Comment:
nit: for consistency, I would prefer: `char pathname[] = "//**null-file-path**//";`